c_cxx/OpenVINO_EP/Linux/squeezenet_classification/README.md

```diff
@@ -10,10 +10,10 @@ The source code for this sample is available [here](https://github.com/microsoft
 
```

Comment:
Above How to Build change 
The source code for this sample is available [here](https://github.com/microsoft/onnxruntime-inference-examples/tree/main/c_cxx/OpenVINO_EP/squeezenet_classification).
to 
The source code for this sample is available [here](https://github.com/microsoft/onnxruntime-inference-examples/tree/main/c_cxx/OpenVINO_EP/**Linux/**squeezenet_classification).

Reply:
Done